patches/components-content_settings-core-common-content_settings.cc.patch

```diff
@@ -1,8 +1,16 @@
 diff --git a/components/content_settings/core/common/content_settings.cc b/components/content_settings/core/common/content_settings.cc
-index 4de6d5608a08863a7948111394758d692af0c0d5..ccfd01c0ecfb789b441d045ebf3fbe581278f087 100644
+index 4de6d5608a08863a7948111394758d692af0c0d5..86625f95ebc8bbf9fac906c8129ecae2aa00f406 100644
 --- a/components/content_settings/core/common/content_settings.cc
 +++ b/components/content_settings/core/common/content_settings.cc
-@@ -179,6 +179,7 @@ bool RendererContentSettingRules::IsRendererContentSetting(
+@@ -93,6 +93,7 @@ constexpr HistogramValue kHistogramValue[] = {
+     {ContentSettingsType::FONT_ACCESS, 71},
+     {ContentSettingsType::PERMISSION_AUTOREVOCATION_DATA, 72},
+     {ContentSettingsType::FILE_SYSTEM_LAST_PICKED_DIRECTORY, 73},
++    BRAVE_HISTOGRAM_VALUE_LIST
```

Comment:
we can get rid of this by overriding ContentSettingTypeToHistogramValue and managing our own list (delegating to the original method for anything not in our list)
We should start our list well above the last chromium value like maybe 1000 or something

Reply:
but also do we even need this?